### PR TITLE
Set up 3D viewing mode

### PR DESCRIPTION
**Pull request by petersbob**
resolving #12

### CODE REVIEW

**Comment by petersbob** [comment]
Very basic 3D --> 2D viewer. 3D coordinate system and a 2D coordinate system that orientates itself based on the axis selected. If the Y axis is selected, slices will be 2D shapes along the Y and Z axis of the 2D viewer.

File: demos/3d_mode.html [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+		'data':'#3090FF'
+	}
+	var params = {} //gui params
+	var view;
+	var equation = {
+		eq: null,
+		parse: function(str){
+			//Takes a string and stores the new equation
+			// In case it's a malformed equation, set eq to null so we know something went wrong
+			this.eq = Parser.parse( str ).toJSFunction( ['x'] );
```
Comment (sambrodriguez) [inline demos/3d_mode.html:26]: Is there a reason for the spaces around the passed in params? It makes me feel dirty :(

Comment (hawkrives) [reply demos/3d_mode.html:26]: (Apologies; Sam rediscovered his github account. You can ignore him.)

Comment (sambrodriguez) [reply demos/3d_mode.html:26]: <3

Comment (petersbob) [reply demos/3d_mode.html:26]: 3d_mode.html is a close copy of 2d_mode.html. The only reason it is formatted that way is that is how 2d_mode.html does it.

**Comment by OmarShehata** [comment]
Looks like a good start! Right now you have X and Y like in the 2D mode. We should have Z there as well. I'll merge this and we can build off of it.

**Pull request merged**